Disable --fix for sort-prop-types

webOS101 opened the pull request:
This addresses #1940 by disabling the fix option for `sort-prop-types` until a better fix to copy the comments is available.

webOS101 commented:
I'd hoped to keep this PR simple, but it required removing quite a bit of code.  Also, there are some failing Travis tests that seem unrelated to this PR.  Finally, not sure why, but I can't seem to execute tests locally.

ljharb commented on lib/rules/sort-prop-types.js:
```diff
@@ -98,18 +95,6 @@ module.exports = {
         return;
       }
 
-      function fix(fixer) {
```
could we keep this as commented code, and do something like `const fix = undefined;` or something, so that all the rest of the code passing `fix` doesn't have to change?

webOS101 replied:
I thought it might be easier to make this easy to revert to restore the missing code when desired.  I tend to not like to keep large swaths of commented out code around, but I can do that if desired.

ljharb replied:
generally i'd agree, but in this case i think it'd be preferable.

ljharb commented on tests/lib/rules/sort-prop-types.js:
```diff
@@ -486,18 +486,7 @@ ruleTester.run('sort-prop-types', rule, {
       column: 5,
       type: 'Property'
-    }],
-    output: [
```
similarly i wonder if we could comment out instead of remove all these `output`s - since they're all correct for when we re-enable the fixer.

webOS101 commented:
I can add the failing test case, commented out.  As mentioned, I was keeping this easy to revert.  I can either add the failing test here or in another PR, or I if you do want me to go the comment-out route, I can easily add the test in.  I jus remembered that `sort-default-props` also uses the same code and probably suffers the same problem, I didn't test it though.

webOS101 commented:
@ljharb I went ahead and changed the removals to comments and added failing (but commented out) tests for sort with comments.  I did not yet disable `--fix` for `jsx-sort-default-props` but think I should probably do that as well in this same PR.  Agree?

ljharb commented:
Yes, I agree. Thanks!

webOS101 commented:
@ljharb Disabled fix for `jsx-sort-default-props`.  I _think_ this is more-or-less ready to go.

ljharb commented:
Looks like there's some tests failing on master :-/
